src/gas/gas-util.ts

```diff
@@ -0,0 +1,104 @@
+import { BN } from 'ethereumjs-util';
+import { query, handleFetch, gweiDecToWEIBN, weiHexToGweiDec } from '../util';
+import {
+  GasFeeEstimates,
+  EthGasPriceEstimate,
+  EstimatedGasFeeTimeBounds,
+  unknownString,
+  LegacyGasPriceEstimate,
+} from './GasFeeController';
+
+const GAS_FEE_API = 'https://mock-gas-server.herokuapp.com/';
+export const EXTERNAL_GAS_PRICES_API_URL = `https://api.metaswap.codefi.network/gasPrices`;
+
+export async function fetchGasEstimates(): Promise<GasFeeEstimates> {
+  return await handleFetch(GAS_FEE_API);
+}
+
+/**
+ * Hit the legacy MetaSwaps gasPrices estimate api and return the low, medium
+ * high values from that API.
+ */
+export async function fetchLegacyGasPriceEstimates(): Promise<LegacyGasPriceEstimate> {
+  const result = await handleFetch(EXTERNAL_GAS_PRICES_API_URL, {
+    referrer: EXTERNAL_GAS_PRICES_API_URL,
+    referrerPolicy: 'no-referrer-when-downgrade',
+    method: 'GET',
+    mode: 'cors',
+    headers: {
+      'Content-Type': 'application/json',
+    },
+  });
+  return {
```

Comment:
Should this be slow, average, fast?

Reply:
I thought about it, but I think the synergy afforded of having gasPriceEstimates.low/medium/high on two of the three valid gas estimate types is better, and the mapping of slow/avg/fast can happen on app side.

Reply:
Makes sense, thanks Brad!